pkg/cvo/cvo_scenarios_test.go

```diff
@@ -2808,7 +2868,7 @@ func verifyAllStatus(t *testing.T, ch <-chan SyncWorkerStatus, items ...SyncWork
 		}
 
 		if !reflect.DeepEqual(expect, actual) {
-			t.Fatalf("unexpected status item %d: %s", i, diff.ObjectReflectDiff(expect, actual))
+			t.Fatalf("unexpected status item %d\nExpected: %#v\nActual: %#v", i, expect, actual)
```

Comment:
is there a reason why you found showing the full expected and actual content was easier to read than the diff?

Reply:
Documented reason in commit message.